Classes/ContentObject/JsonDecoder.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
```

Comment:
Add header comment

Reply:
yeah, forgot about that! :) thank you! :) fixed in next commit